export.html

```diff
@@ -0,0 +1,288 @@
+
+<p>Select the OME-TIFF version to export, from the drop-down in the bottom left.
+</p>
+
+<p>Click <span class="bold">Export</span> to create and download the OME-TIFF as a ZIP archive.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/exportOMETIFFInsight.jpg" class="align-left" src="images/exportOMETIFFInsight.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<p>Click <span class="bold">Create</span> to generate the OME-TIFF using the latest version.</p>
+
+<p>The OME-TIFF is attached to the image as a ZIP archive.</p>
+
+<p>Click <span class="bold">Download</span> in the Activity Window to download the archive.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/exportOMETIFFWeb.jpg" class="align-center" src="images/exportOMETIFFWeb.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">Select the appropriate format to export the image as a JPEG, PNG or TIFF.</p>
+
+<p>In OMERO.insight, use File Finder in the Download window to select the destination.</p>
+
+<p>In OMERO.web the file will be downloaded according to your browser convention.</p>
+
+<h3>OMERO.insight</h3>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/exportFormatsInsight.jpg" class="align-left" src="images/exportFormatsInsight.jpg" style="width: 75%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/exportFormatsWeb.jpg" class="align-center" src="images/exportFormatsWeb.jpg" style="width: 80%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="note">
+<p class="note-title">Note</p>
+<p class="note-text">When an image or dataset is downloaded as a JPEG, PNG or TIFF, individual images for each channel and a combined-channel image is generated for the selected Z section or time point for the image(s). All generated images are contained in a ZIP archive.</p>
```

Comment:
'... individual images for each channel and a combined-channel image is generated...' -> '... individual images for each channel and a combined-channel image are generated...'